python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -1,200 +1,265 @@
-from dataclasses import dataclass, field
 from datetime import datetime
-from enum import Enum, auto
-from typing import Any
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, Field
 
 DATAHUB_DATE_FORMAT = "%Y%m%d"
 
 
-@dataclass
-class CatalogueMetadata:
-    name: str
-    description: str
-    owner: str
-    tags: list[str] = field(default_factory=list)
+class DatabaseStatus(Enum):
```

Comment:
Think i agree and this isn't adding much other than potential confusion. Is probably a bit of leftover from the previous iteration's schema where we were concerned re. the status of a piece of data that had been uploaded, and [this description](https://github.com/ministryofjustice/data-platform/blob/50a46dab56ecabc5e1f1ccae6c02e546fb6efbd7/python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py#L132) kind of backs that up (description not aligning with enum values). I'm not sure why anyone would need to catalogue data or a database that was draft status.

Guess it could play somehow into gated registration to the catalogue but that wouldn't be how it's currently being used and not database specific

Reply:
Yeah, that makes sense. I would feel better revisiting this once we are actually implementing gated registration - we are not there yet